src/tests/fake/utils.ts

```diff
@@ -0,0 +1,5 @@
+export type FakeSearchResults = {
+    page_count?: number;
+    per_page?: number;
+    total_count?: number;
```

Comment:
Shouldn't these properties always be defined?

Reply:
Ah no, which points to a naming mistake. The idea here was that it would work as a base model for allowing the fake functions to take optional manual overrides for the search result data returned by nock. `fakeSearchResultOptions`?